docs/EVG.md

```diff
@@ -1864,3 +1872,20 @@ guidelines:
    jurisdiction as specified in Section 9.2.4.  The stated address of the organisation 
    combined with the organization name SHALL NOT be the only information used to 
    disambiguate the organisation.
+   
+**LEI**: The Legal Entity Identifier (LEI) as specified in the ISO 17442 and registered
+   in the Global LEI Index.  
```

Comment:
So it's ambiguous here as to what "registered in the Global LEI Index" means. It implies that an accredited-by-GLEIF LOU can withhold data from the Global LEI System, and thus some additional requirement is required.

If the changes to LEI are made within the definitions (e.g. does it have to be an LEI defined by an LOU accredited by GLEIF/the LEI ROC), then I think this entire first sentence may be unnecessary.

Alternatively, if we want to avoid normative requirements within the definition, such that LEI is merely the number that conforms to ISO 17442, then this is where we'd introduce the requirement that the LEI be issued by an LOU recognized by the Global Legal Entity Identifier Foundation.

No definition for "who" the foundation is is necessary.